ingest/scripts/compare_hashes.py

```diff
@@ -265,6 +283,21 @@ def main(
         else:
             logger.info(f"{text}: {len(value)}")
 
+    if len(ingested_insdc_accessions) > 0:
```

Comment:
Maybe it would be cleaner to use a different variable and not mutably change one without changing the name.